src/Tribe/Log/Service_Provider.php

```diff
@@ -164,7 +164,7 @@ public function enable() {
 	/**
 	 * Disables the logging functions.
 	 *
-	 * @since TBD
+	 * @since 4.12.15
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: dcb92ce6070892036783bd78f76e5cda</sub>